### ISSUE TITLE
 More friendly error message if not running VS as Admin

### ISSUE THREAD

**Issue by dazinator** [dazinator/DnnPackager#39]
When running the `Install-Module` command, DnnPackager uses the `WebAdministration` library to query IIS to get info about the Dnn website. However this errors if not running VS with sufficient permissions. Raise a friendlier error message when this is the case asking the user to "please ensure you are running VS as Administrator"

As a secomdary task, find out if its possible to set things up.in IIS in such a way that running VS as Admin is not necessary.

**Comment by nvisionative** [comment]
Good idea on the first item!  Regarding the secondary task, that is worth looking into for sure.  Many of us use MakeDnnSite for very fast local DNN instance setup, so whatever you discover may have to be a tweak post initial setup via MakeDnnSite.

**Comment by dazinator** [comment]
Interestring.. not heard of makednnsite before! Sounds handy!

**Comment by nvisionative** [comment]
I just sent you the MakeDNNSite app - hope it helps!
